docs/tasks/administer-cluster/pvc-protection.md

```diff
@@ -6,18 +6,16 @@ title: Persistent Volume Claim Protection
 ---
 
 {% capture overview %}
-{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
+{% assign for_k8s_version="v1.10" %}{% include feature-state-beta.md %}
 
-As of Kubernetes 1.9, persistent volume claims (PVCs) that are in active use by a pod can be protected from pre-mature removal.
+As of Kubernetes 1.10, persistent volume claims (PVCs) that are in active use by a pod can be protected from pre-mature removal.
 
 {% endcapture %}
 
 {% capture prerequisites %}
 
-- A v1.9 or higher Kubernetes must be installed.
-- As PVC Protection is a Kubernetes v1.9 alpha feature it must be enabled:
-1. [Admission controller](/docs/admin/admission-controllers/) must be started with the [PVC Protection plugin](/docs/admin/admission-controllers/#persistent-volume-claim-protection-alpha).
-2. All Kubernetes components must be started with the `PVCProtection` alpha features enabled.
+- A v1.10 or higher Kubernetes must be installed.
```

Comment:
Again not technically correct since it was available in 1.9.  Maybe the requirement should be that the feature is enabled in a supported release?